components/ingest-service/test/chef_action.js

```diff
@@ -58,7 +58,7 @@ describe("/events/chef/action", function () {
           return helpers.ActionRefresh().then(function(response) {
             return chakram.get(actionsUrl()).then(function(response) {
               expect(response).to.have.status(200);
-              expect(response.body.hits.total).to.equal(0);
+              expect(response.body.hits.total.value).to.equal(0);
```

Comment:
Elasticsearch 7 response changed to return an object for the total field with the value inside.